modules/configuring-a-recommended-accelerator-for-notebook-images.adoc

```diff
@@ -9,25 +9,22 @@ To help you indicate the most suitable accelerators to your data scientists, you
 .Prerequisites
 * You have logged in to {openshift-platform}.
 * You have the `cluster-admin` role in {openshift-platform}.
-* You have configured an ImageStream that contains your notebook image settings.  
+* You have existing notebook images in your deployment.
 
 .Procedure
-. In the {openshift-platform} web console, in the *Administrator* perspective, click *Builds* -> *ImageStreams*.
-. Click the ImageStream that you want to a recommended accelerator tag to. 
+. From the {productname-short} dashboard, click *Settings* -> *Notebook images*.
```

Comment:
The button is  *Notebook image settings*

Reply:
Fixed.